docs/app/data/css-page.json

```diff
@@ -264,6 +264,21 @@
                     "title": "UX Aspects Icons",
                     "component": "CssUxIconsComponent"
                 },
+                {
+                    "title": "Icons Documentation",
```

Comment:
Might want to reconsider this title, every section is "documentation" so this seems a bit redundant. Also, as this section is an Angular component should this be in the components section rather than CSS? thoughts @donna-marijne?

Reply:
Agreed, we should establish an "Icons" category under "Components," and the section should be named "Icon." The documentation section containing the list of icons should be renamed to "Icon Set."

Reply:
Moved to an Icons Section under components and renamed list of icons to Icon Set